server/src/test/java/org/elasticsearch/action/admin/cluster/configuration/TransportClearVotingConfigExclusionsActionTests.java

```diff
@@ -214,4 +242,23 @@ public ActionResponse.Empty read(StreamInput in) {
             }
         };
     }
+
+    static class FakeReconfigurator extends Reconfigurator {
```

Comment:
nit Maybe this one can be refactored with TransportAddVotingConfigExclusionsActionTests.java's FakeReconfigurator?